src/order/symm_diff.lean

```diff
@@ -21,6 +21,7 @@ The symmetric difference is the addition operator in the Boolean ring structure
 ## Main declarations
 
 * `symm_diff`: the symmetric difference operator, defined as `(A \ B) ⊔ (B \ A)`
+* `equiv.symm_diff`: Symmetric difference by `a` as an `equiv`.
```

Comment:
This is inaccurate, right? The equiv isn't anywhere right now

Reply:
Argh, copy-paste 😢